spec/sec-properties-of-the-regexp-prototype-object-patch.html

```diff
@@ -122,121 +107,71 @@ <h1><ins>GetStringIndex ( _S_, _Input_, _e_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getmatchresult" aoid="GetMatchResult">
-      <h1><ins>GetMatchResult ( _S_, _startIndex_, _endIndex_, _matchOptions_ )</ins></h1>
-      <p>The abstract operation GetMatchResult with arguments _S_, _startIndex_, _endIndex_, and _matchOptions_ performs the following steps:</p>
+    <emu-clause id="sec-getmatchstring" aoid="GetMatchString">
+      <h1><ins>GetMatchString ( _S_, _match_ )</ins></h1>
+      <p>The abstract operation GetMatchResult with arguments _S_ and _match_ performs the following steps:</p>
       <emu-alg>
       1. Assert: Type(_S_) is String.
-      1. Assert: _startIndex_ is an integer index into _S_.
-      1. Assert: _endIndex_ is an integer index into _S_ that is greater than or equal to _startIndex_.
-      1. Assert: _matchOptions_ is a MatchOptions Record.
-      1. If _matchOptions_.[[Capture]] is `"indices"`, then
-        1. Return CreateArrayFromList(&laquo; _startIndex_, _endIndex_ &raquo;).
-      1. Else,
-        1. Return the portion of _S_ between offset _startIndex_ inclusive and offset _endIndex_ exclusive.
+      1. Assert: _match_ is a Match Record.
+      1. Assert: _match_.[[StartIndex]] is an integer index into _S_.
```

Comment:
is "index into" something that's already used elsewhere? (as opposed to, eg, asserting it's >= 0 and < S's length)

Reply:
It's a phrase used quite a bit in the existing RegExp portions of the spec.